.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+Contributing Guidelines
+--------------------------------------------------
+
+This document provides general guidelines about how to contribute to the project. Keep in mind these important things before you start contributing.
+
+### Asking Questions
+
+We do not use github issues for general library support. We think this questions should be posted on stack overflow using [InstantSearch](http://http://stackoverflow.com/questions/tagged/InstantSearch) tag.
+
+### Reporting issues
+
+* Use [github issues](https://github.com/algolia/InstantSearch/issues) to report a bug.
+* Before creating a new issue:
+  * Make sure you are using the [latest release](https://github.com/algolia/InstantSearch/releases).
+  * Check if the issue was [already reported or fixed](https://github.com/algolia/InstantSearch/issues?utf8=%E2%9C%93&q=is%3Aissue). Notice that it may not be released yet.
+  * If you found a match add a brief comment "I have the same problem" or "+1". This helps prioritize the issues addressing the most common and critical first. If possible add additional information to help us reproduce and fix the issue. Please use your best judgement.    
+* Reporting issues:
+  * Please include the following information to help maintainers to fix the problem faster:
+    * Xcode version you are using.
+    * iOS version you are targeting.
```

Comment:
As Search Client is a common Swift library, I think it's better to indicate _iOS/macOS/tvOS version_